catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -337,6 +344,12 @@
                 false /* tokenized */,
                 true /* multivalued */,
                 STRING_TYPE));
+        descriptors.add(new AttributeDescriptorImpl(Validation.VALIDATORS,
+                true /* indexed */,
```

Comment:
I agree with Richard.  I do not think the validators active on the system would often change after initial setup.  That field would always have the same list if that is the common case.  

A list for validators that gave warnings and another for ones that gave errors might be useful when doing federated queries but not sure yet.  Would have to consider if a version is included with the validator name.